demo/omni-slider.js

```diff
@@ -269,36 +263,19 @@ var Slider = function () {
 
       // Get handle element node not the child nodes
       // If this is a child of the parent try to find the handle element
-      if (this.isIE8) {
-        this.dragObj.elNode = window.event.srcElement;
-
-        while (this.dragObj.elNode.getAttribute('class').indexOf('handle') < 0) {
-          this.dragObj.elNode = this.dragObj.elNode.parentNode;
-        }
-      } else {
-        this.dragObj.elNode = event.target;
+      this.dragObj.elNode = event.target;
 
-        while (!this.dragObj.elNode.classList.contains('handle')) {
-          this.dragObj.elNode = this.dragObj.elNode.parentNode;
-        }
+      while (!this.dragObj.elNode.classList.contains('handle')) {
+        this.dragObj.elNode = this.dragObj.elNode.parentNode;
       }
 
       // Direction where the slider control is going
-      if (this.isIE8) {
-        this.dragObj.dir = this.dragObj.elNode.getAttribute('class').indexOf('handle-left') >= 0 ? 'left' : 'right';
-      } else {
-        this.dragObj.dir = this.dragObj.elNode.classList.contains('handle-left') ? 'left' : 'right';
-      }
+      this.dragObj.dir = this.dragObj.elNode.classList.contains('handle-left') ? 'left' : 'right';
 
       // Get cursor position wrt the page
-      if (this.isIE8) {
-        x = window.event.clientX + document.documentElement.scrollLeft + document.body.scrollLeft;
-        y = window.event.clientY + document.documentElement.scrollTop + document.body.scrollTop;
-      } else {
-        // If touch screen (event.touches) and if IE11 (pageXOffset)
-        x = (typeof event.clientX !== 'undefined' ? event.clientX : event.touches[0].pageX) + (window.scrollX || window.pageXOffset);
-        y = (typeof event.clientY !== 'undefined' ? event.clientY : event.touches[0].pageY) + (window.scrollY || window.pageYOffset);
-      }
+      // If touch screen (event.touches) and if IE11 (pageXOffset)
+      x = (typeof event.clientX !== 'undefined' ? event.clientX : event.touches[0].pageX) + (window.scrollX || window.pageXOffset);
```

Comment:
this is compiled version from babel but im sure this happens on the file under `src` too